packages/client/hmi-client/src/components/workflow/ops/model-edit/tera-model-edit.vue

```diff
@@ -116,6 +116,7 @@ import TeraModelTemplateEditor from '@/components/model-template/tera-model-temp
 import TeraNotebookJupyterInput from '@/components/llm/tera-notebook-jupyter-input.vue';
 
 import { KernelSessionManager } from '@/services/jupyter';
+import { getModelConfigurationById } from '@/services/model-configurations';
```

Comment:
Why not create a new method `getModelFromModelConfigurationById`?

Reply:
good point